cmd/pulumictl/copyright/cli.go

```diff
@@ -189,6 +198,21 @@ func (c *checker) findAllFiles() ([]string, error) {
 	lines := splitLines(string(out))
 	var files []string
 	for _, line := range lines {
+		// Infuriatingly -x in 'git ls-files' only applies to untracked files so
+		// we need to post-filter.
+		var matched bool
+		for _, pattern := range c.exclude {
+			matched, err = dstar.PathMatch(pattern, line)
+			if err != nil {
+				return nil, err
+			}
+			if matched {
```

Comment:
My FP brain gets broken a little every time by break and continue but I think it does what it needs to do here.

Reply:
> My FP brain gets broken a little every time by break and continue but I think it does what it needs to do here.

😜 Sounds like I need to learn more than you have to unlearn.